packages/dart.rb

```diff
@@ -0,0 +1,24 @@
+require 'package'
+
+class Dart < Package
+  version '4.4.0.148'
+
+    source_url ({
+    i686: "https://www.dropbox.com/s/d1omw087ilkh5oq/jdk1.8.0_25_i686.tar.gz?dl=0",
+    x86_64: "https://www.dropbox.com/s/ykw2jt797cpghfy/jdk1.8.0_65_x86_64.tar.gz?dl=0"
+  })
```

Comment:
@thedamian I'm always trying not to complain about indentation, but this is so visible 😀